Sources/Shared/Combine/VimeoClient+Publisher.swift

```diff
@@ -0,0 +1,36 @@
+//
+//  VimeoClient+Publisher.swift
+//  VimeoNetworking
+//
+//  Created by Rogerio de Paula Assis on 11/17/19.
+//  Copyright © 2019 Vimeo. All rights reserved.
+//
+
+import Foundation
+import Combine
+
+extension VimeoClient {
+
+    /// Returns a publisher that wraps a decodable request for a given URLConvertible type.
```

Comment:
Would it make sense to specify `EndpointType` here instead of `URLConvertible` [_sic_]?